cod/iw4x/egg-iw4x-mw2.json

```diff
@@ -121,13 +121,23 @@
         },
         {
             "name": "Server Name",
-            "description": "Sets the server hostname.\r\n SERVER  COLORS \r\n^1 = Red , ^2 = Green, ^3 = Yellow, ^4 = Blue, ^5 = Cyan, ^6 = Pink, ^7 = White, ^8 = Depends on the team colors playing.,  ^9 = Grey, ^0 = Black, ^: = Rainbow colors.",
+            "description": "Sets the server hostname. Name needs to be in \" \" like as example \"Pterodactyl IW4X\"\r\n SERVER  COLORS \r\n^1 = Red, ^2 = Green, ^3 = Yellow, ^4 = Blue, ^5 = Cyan, ^6 = Pink, ^7 = White, ^8 = Depends on the team colors playing.,  ^9 = Grey, ^0 = Black, ^: = Rainbow colors.",
```

Comment:
Can the quotes not be included in the output in the config parser, instead of needing the user to enter them ?